packages/ddp-client/package.js

```diff
@@ -32,6 +32,15 @@ Package.onUse((api) => {
 
   api.use('reload', 'client', { weak: true });
 
+  // If the application is using sockjs-shim, then it needs to be loaded
+  // before ddp-client so that it can polyfill global.SockJS. However, we
+  // don't want to force sockjs-shim to be loaded, since ddp-client is
+  // part of an isopacket used by the Meteor command-line tool, and the
+  // webapp package (used by server-render, which is used by sockjs-shim)
+  // is not safe to load as part of an isopacket, because it calls
+  // Fiber.yield during package initialization. Hence the weakness here.
```

Comment:
This comment should probably indicate whether or not someone else needs to depend on `sockjs-shim` for this to work.

(I guess they do, since it's in `meteor-base` now)